resources/analysisTools/indelCallingWorkflow/confidenceAnnotation_Indels.py

```diff
@@ -213,14 +217,12 @@ def main(args):
         penalties = ""
         infos = []
 
-        if args.no_control:
+        if args.no_control or args.refgenome[0] == 'GRCh38' or args.skipREMAP:
```

Comment:
- I have observed the inconsistency, but I have left `hs37d5` in for legacy reasons and worked around it.
- `hs37d5` is like a version number for 1000 genome reference genome of GRCh37. 
- Yes, it could be improved, I will look into these.

Reply:
You could for instance add two functions grch37 and grch38 that each return 'true' or 'false' (0/1 in Perl :P) on match to one or multiple equivalent identifiers. Then you could also match hg37, hg19, hg38, hs37d5, GRCh37, GRCh38, etc. all equivalently.

Reply:
I just looked at the hg38 branch and saw that we had a related discussion about the used identifiers there. Probably the ideal would be to test these identifiers already in the Java/Groovy code, maybe even map from aliases to the canonical names used everywhere else in the code (e.g. hg19->hs37d5). However, maybe it is sufficient to have the identifiers documented well in the XML and maybe the README. 

The room for improvement of the if/else spaghetti code remains a valid point, though.